legal-api/src/legal_api/resources/v2/business/business_resolutions.py

```diff
@@ -44,16 +44,17 @@ def get_resolutions(identifier, resolution_id=None):
 
     # return the matching resolution
     if resolution_id:
-        resolution, msg, code = _get_resolution(legal_entity, resolution_id)
+        resolution, msg, code = _get_resolution(business, resolution_id)
         return jsonify(resolution or msg), code
 
     resolution_list = []
 
     resolution_type = request.args.get("type")
     if resolution_type:
-        resolutions = Resolution.find_by_type(legal_entity.id, resolution_type.upper())
+        business_id = business.id if business.is_legal_entity else business.legal_entity_id
```

Comment:
this function will only ever be called for non-firms.  so you can always use `business.id`.  don't need this line.

Reply:
updated